honeybee_radiance/sensorgrid.py

```diff
@@ -283,6 +296,22 @@ def room_identifier(self):
     def room_identifier(self, n):
         self._room_identifier = typing.valid_string(n)
 
+    @property
+    def group_identifier(self):
+        """Get or set text for the grid group identifier to which this SensorGrid belongs.
+
+        This will be used in the write to radiance folder method to write all the grids
+        with the same group identifier under the same subfolder.
+
+        If None, the grid will be written to the root of grids folder.
+        """
+        return self._group_identifier
+
+    @group_identifier.setter
+    def group_identifier(self, n):
+        identifier_key = '/'.join(typing.valid_string(key) for key in n.split('/'))
```

Comment:
You should use `valid_rad_string` here instead of `valid_string`.  The former will only check for the illegal radiance characters and not the eillegal characters for both E+ and Radiance.

I think the ability to create sub-sub-folders by using multiple `/` is a good one but I think you should make a note of this capability in the `group_identifier` docstring.